go/private/repositories.bzl

```diff
@@ -134,6 +138,14 @@ def go_rules_dependencies():
       # importpath = "google.golang.org/genproto",
   )
 
+  # GRPC Gateway
+  _maybe(git_repository,
+      name = "com_github_grpc_ecosystem_grpc_gateway",
+      remote = "https://github.com/Mistobaan/grpc-gateway", # FIXME once 266 is merged
```

Comment:
What is 266? The issue with that number in `github.com/grpc-ecosystem/grpc-gateway` is unrelated.

This should be merged before proceeding with this PR. We should depend on the real repo.